### DISCUSSION TITLE
When exporting, please include the current page and all subpages

### DISCUSSION THREAD

**Discussion by aliuq** [toeverything/AFFiNE#6408] [Ideas]
### Description

I have a scenario where I need to export content and share it with others for offline use. The exported html cannot contain subpages.

Similarly, when I share a document that contains a subpage, the subpage cannot be accessed in the shared link.

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by JimmFly** [comment]
What do you think about this? @hwangdev97

**Comment by ldehner** [comment]
Is there any option for this yet @JimmFly? Maybe at least for snapshots to "move" between workspaces?

**Reply by ldehner** [reply]
Currently this experience is super frustrating. 

1. It gives me a little headache that there is no possibility for mass export 
2. Even when I want to move between workspaces its super painful
3. When I export a doc with linked docs inside, the linked docs are not exported. After I manually exported the linked docs and then imported them they are not recognised. **I have to manually delete the links and recreate them. Very annoying.**

**Reply by zetashift** [reply]
> It gives me a little headache that there is no possibility for mass export

This is the biggest blocker for me for using AFFiNE as my main tool for note taking and organizing. Mass export to markdown or CSV or JSON, is a must to not be locked into a single piece of software.
